test/functional/cmdline_options_tester/src/Tokenizer.java

```diff
@@ -81,30 +81,36 @@ private String token() throws Exception {
          * otherwise it will ends up returning an empty argument to be ignored in forming the command array.
          *
          * Three cases are taken into account in the while loop:
-         * 1) Strings in double-quotes (a argument starting with double-quotes(")) are completely extracted from the command string,
-         *    whether white spaces in the argument exist or not. Double-quotes themselves are discarded after extraction when scanning forward.
-         * 2) Normal arguments (no double-quotes occur in the argument) are split up and extracted with white spaces between arguments.
+         * 1) Strings in double-quotes/single-quotes (a argument starting with double-quotes(") or single-quotes(')) are completely extracted from the command string,
```

Comment:
While you're changing this comment can you also fix the grammar, please (e.g. 'an argument')?

Reply:
Fixed!